Permissions/Core/MvxPermissions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Plugin.Permissions;
+using Plugin.Permissions.Abstractions;
+
+namespace MvvmCross.Plugins.Permissions
+{
+    public class MvxPermissions : IMvxPermissions
+    {
+        public Task<bool> ShouldShowRequestPermissionRationaleAsync(Permission permission)
```

Comment:
There is nothing wrong with this. No need to await it here, you can just pass the task along.